share/completions/ansible.fish

```diff
@@ -0,0 +1,106 @@
+# fish 2.2.0 does not include native snippet support. Upgrade to fish >= 2.3.0
+# or append the following code to your ~/.config/fish/config.fish.
+
+# for file in ~/.config/fish/conf.d/*.fish
+#     source $file
+# end
+# ansible
+complete -c ansible -s a -l args -r -f -d "module arguments"
+complete -c ansible -l ask-vault-pass -f -d "ask for vault password"
+complete -c ansible -s B -l background -r -f -d "run asynchronously, failing after X seconds (default=N/A)"
+complete -c ansible -s C -l check -f -d "don't make any changes; instead, try to predict some of the changes that may occur"
```

Comment:
This should be a bit shorter. Something like "Just check, don't make any changes"?

Reply:
(Thats a general thing - descriptions should be short, so we can fit more on a screen)